resources/views/forum/threads/show.blade.php

```diff
@@ -27,24 +27,44 @@
             <a class="btn btn-link btn-block" href="{{ route('forum') }}"><i class="fa fa-arrow-left"></i> Back</a></a>
         </div>
         <div class="col-md-9">
-            <h1>{{ $title }}</h1>
-            <hr>
+            @include('forum.threads._header')
+            <hr class="header-divider">
```

Comment:
Can we try adding the `hr` right after the title? I think it will be more in line with the other places which have a `h1` which have the `hr` right after it.